asserters_test.go

```diff
@@ -0,0 +1,386 @@
+package safecast_test
+
+import (
+	"errors"
+	"math"
+	"strings"
+	"testing"
+
+	"github.com/ccoveille/go-safecast"
+)
+
+// The tests in conversion_all_test.go are the ones that are not architecture dependent
+// The tests in conversion_64bit_test.go complete them for 64-bit systems
+//
+// This architecture dependent file covers the fact, you can reach
```

Comment:
incomplete comment. did you mean:
>// This architecture dependent file covers the fact, you can reach a higher value with int and uint
// on 64-bit systems, but you will get a compile error on 32-bit.
// This is why it needs to be tested in an architecture dependent way.